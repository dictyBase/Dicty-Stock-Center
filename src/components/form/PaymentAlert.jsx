import React, { Component, PropTypes } from 'react'
import 'styles/core.scss'

export default class PaymentAlert extends Component {
    displayName = 'payment information'

    static propTypes = {
        paymentInfoLink: PropTypes.string.isRequired
    }

    static defaultProps = {
        paymentInfoLink: '/stockcenter/payments/information'
    }

    render() {
        return (
            <div>
                <div className="alert alert-info">
                    <p><strong>Credit Card: </strong>
                        Secure payment when billed
                    </p>
                    <p><strong>Wire Transfer: </strong>
                        Northwestern bank information will be emailed
                    </p>
                    <p><strong>PO: </strong>
                        Add PO number if available or send ASAP to&nbsp;
                        <u><a href="mailto:dictystocks@northwestern.edu" target="_top">
                            dictystocks@northwestern.edu
                        </a></u>
                    </p>
                </div>
                <div className="alert alert-info">
                    <p>For full payment information please click&nbsp;
                        <u><a href={ this.props.paymentInfoLink } target="_blank">here</a></u>
                    </p>
                </div>
            </div>
        )
    }
}
