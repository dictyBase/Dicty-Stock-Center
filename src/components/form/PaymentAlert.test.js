import React from 'react'
import renderer from 'react-test-renderer'
import PaymentAlert from './PaymentAlert'

test('matching a snapshot of PaymentAlert', () => {
    const component = renderer.create(
    <PaymentAlert />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
