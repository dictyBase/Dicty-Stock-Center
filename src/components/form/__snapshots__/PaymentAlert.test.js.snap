// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot of PaymentAlert 1`] = `
<div>
  <div
    className="sc-ifAKCX iMBukv"
  >
    <p>
      <strong>
        Credit Card: 
      </strong>
      Secure payment when billed. Incurs a 
      <strong>
        3.5% service fee.
      </strong>
    </p>
    <p>
      <strong>
        Wire Transfer: 
      </strong>
      Northwestern bank information will be emailed
    </p>
    <p>
      <strong>
        PO: 
      </strong>
      Add PO number if available or send ASAP to 
      <u>
        <a
          href="mailto:dictystocks@northwestern.edu"
          target="_top"
        >
          dictystocks@northwestern.edu
        </a>
      </u>
    </p>
  </div>
  <div
    className="sc-ifAKCX iMBukv"
  >
    <p>
      For full payment information please click 
      <u>
        <a
          href="/stockcenter/payments/information"
          target="_blank"
        >
          here
        </a>
      </u>
    </p>
  </div>
</div>
`;
